src/hooks/useSignMessage/useSignMessage.ts

```diff
@@ -318,8 +339,9 @@ const useSignMessage = ({
         // Unfortunately that isn't possible, because isValidSignature only takes a bytes32 hash; so to sign this with
         // a personal message, we need to be signing the hash itself as binary data such that we match 'Ethereum signed message:\n32<hash binary data>' on the contract
 
-        const sig = await (toSign.type === 'eth_signTypedData_v4' ||
-        toSign.type === 'eth_signTypedData'
+        const isTypedData = toSign.type === 'eth_signTypedData_v4' || toSign.type === 'eth_signTypedData'
```

Comment:
'isTypedData' is already declared in the upper scope, maybe name it something else 🙄

Reply:
you're right, I thought that it had been removed. I'll just pass isTypedData from upstream